scripts/start.sh

```diff
@@ -64,6 +64,9 @@ fi
 
 docker compose --env-file "/concourse/.concourse.env" up -d
 
+# Download and install the fly CLI from local Concourse instance
+curl -o /usr/local/bin/fly -k 'https://localhost:443/api/v1/cli?arch=amd64&platform=linux' && sudo chmod +x /usr/local/bin/fly
```

Comment:
no need to sudo here

Reply:
I would also probably add sleep 5 before download, just to give a bit more time to concourse for startup, as otherwise download might fail if concourse is not fully started yet

Reply:
also, no need for :443, as that's default port

Reply:
Yes